src/Enums/AvailableNewsletter.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Enums;
+
+enum AvailableNewsletter: string
+{
+    case NEWS = "news";
+    case MEETUPS = "meetups";
+
+    public static function values(): array
+    {
+        $cases = collect(AvailableNewsletter::cases());
+
+        return $cases->map(
+            fn(AvailableNewsletter $enum) => $enum->value,
+        )->toArray();
```

Comment:
```suggestion
        return array_map(self::cases(), fn(AvailableNewsletter $enum): string => $enum->value);
```

This should be quicker.